lib/iris/tests/__init__.py

```diff
@@ -421,7 +421,7 @@ def assertDataAlmostEqual(self, data, reference_filename, **kwargs):
                 stats = json.load(reference_file)
                 self.assertEqual(stats.get('shape', []), list(data.shape))
                 self.assertEqual(stats.get('masked', False),
-                                       isinstance(data, ma.MaskedArray))
+                                 np.ma.is_masked(data))
```

Comment:
@djkirkham We have a soft coding standard where we use `ma` instead of `np.ma`, see [here](https://github.com/SciTools/iris/pull/2477/files#diff-350ecfa750b99028f16446216ccbcfdbL68).

If it's not too much trouble, could you adopt this please, thanks.